shop.go

```diff
@@ -51,7 +51,7 @@ type Shop struct {
 	PrimaryLocationId               int64      `json:"primary_location_id"`
 	Timezone                        string     `json:"timezone"`
 	IanaTimezone                    string     `json:"iana_timezone"`
-	ForceSSL                        bool       `json:"force_ssl"`
+	ForceSSL                        bool       `json:"force_ssl"` // deprecated in version 2021-07, always `true` thereafter
```

Comment:
Same as previous comment. I'd rather let Shopify dictate when each field should be used.

Reply:
Ultimately Shopify is in control over these, however I wanted to add the comments so that our library consumers can have more clarity since this library supports many versions.

Reply:
In the sake of consistency I've removed those comments.